suites/squid/rados/tier-2_rados_test-stretch-mode-upgrade.yaml

```diff
@@ -0,0 +1,264 @@
+# Suite contains tests related to election strategy and Stretched mode
+# Use cluster-conf file: conf/quincy/rados/stretch-mode-host-location-attrs.yaml
+# Stretch mode tests performing upgrade
+
+# This test case is Openstack only and cannot be run in Baremetal env due to test constrains.
+# Stretch mode deployment in BM is run by suite : suites/squid/rados/deploy-stretch-cluster-mode.yaml
+
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Cephadm Bootstrap with apply-spec
+      desc: Apply spec in Bootstrap with host location attributes
+      module: test_bootstrap.py
+      polarion-id: CEPH-83575289
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          rhcs-version: 5.3
```

Comment:
We should change this to 6.1 to maintain n+2 upgrades

Reply:
Made the changes as per the comments.